src/internal_events/elasticsearch.rs

```diff
@@ -16,3 +18,36 @@ impl InternalEvent for ElasticsearchEventEncoded {
         counter!("processed_bytes_total", self.byte_size as u64);
     }
 }
+
+#[derive(Debug)]
+pub struct ElasticSearchResponseError<'a> {
```

Comment:
The convention elsewhere in this file appears to be `Elasticsearch` and not `ElasticSearch` with the middle `S`.

Reply:
I missed this one 🤦‍♂️ thanks